src/server/service/kv.rs

```diff
@@ -369,7 +400,7 @@ impl<T: RaftStoreRouter<E::Local> + 'static, E: Engine, L: LockManager, F: KvFor
 
     fn coprocessor(&mut self, ctx: RpcContext<'_>, req: Request, sink: UnarySink<Response>) {
         forward_unary!(self.proxy, coprocessor, ctx, req, sink);
-        let begin_instant = Instant::now_coarse();
+        let begin_instant = Instant::now();
```

Comment:
Why change this here? It seems the usages are not consistent for example the snapshot duration calculation.

Reply:
now_coarse can only provides 1~10ms precision (depending on the kernel config), which is sometimes not accurate enough for us.